onnxruntime/core/providers/coreml/builders/impl/reduction_op_builder.cc

```diff
@@ -128,6 +132,17 @@ bool ReductionOpBuilder::IsOpSupportedImpl(const Node& node, const OpBuilderInpu
       (node.OpType() == "ReduceMax" || node.OpType() == "ReduceMin" || node.OpType() == "ReduceProd")) {
     return false;
   }
+
+#if defined(TARGET_OS_IOS) && defined(TARGET_CPU_X86_64)
+  // to pass https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1563483&view=logs&j=f7cc61a9-cc70-56e7-b06c-4668ca17e426
+  // ReductionOpTest.ReduceSum_half_bert
```

Comment:
nit: comment would be more helpful to developers if it provides a 'why' or 'what'. the build url is only going to be valid for a few days so an explicit explanation is better. 

e.g. 'skip test on iOS simulator because...'

Reply:
Sure, Will improve the comment in the next PR.